CHANGELOG.md

```diff
@@ -22,8 +22,8 @@
 
 **BREAKING CHANGES:**
 
-While the Pub/Sub library is not GA, and breaking changes are to be expected, we
-are close enough to a GA release that we think highlighting them is important.
+While this is the first GA release of the Pub/Sub library is not GA, and
+breaking changes were to be expected, we think highlighting them is important.
```

Comment:
While this is the first GA release, it is not GA??

Reply:
Fixed, PTAL.